### ISSUE TITLE
Getting a Error: Purging failed with the status code 200.

### ISSUE THREAD

**Issue by motion-work** [ayeressian/bunnycdn-storage-deploy#27]
I am getting a `Error: Purging failed with the status code 200.`

<img width="690" alt="Screenshot 2023-01-23 at 19 29 26" src="https://user-images.githubusercontent.com/11538632/214120500-8b2de787-a084-4cf6-b1d3-1078f7c2edd6.png">

here is my config:
```yml
- name: Deploy to BunnyCDN
  uses: ayeressian/bunnycdn-storage-deploy@v1.1.1
  with:
    source: "./output/public"
    storageZoneName: my-storage-zone-name
    accessKey: "${{ secrets.PROD_STORAGE_KEY }}"
    pullZoneId: 1156109
    pullZoneAccessKey: "${{ secrets.PROD_PULL_ZONE_KEY }}"
    remove: "true"
    purge: "true"
```

anyone can help?

**Comment by ayeressian** [comment]
Because according to their documentation https://docs.bunny.net/reference/pullzonepublic_purgecachepostbytag  there should not be 200 response code. The successful purging should be 204.

**Comment by ayeressian** [comment]
Their backend is inconsistent with their doc.

**Comment by ayeressian** [comment]
Unfortunately, I don't work in bunnycdn and the only reference point I've got is their doc. One thing that we can do is to log the response body to see if there is additional info. To do that there needs to be a new debug tag that you can reference in your action.

**Comment by travis-tap-3d** [comment]
I'm seeing this too; did anyone ever figure this out?

**Comment by ayeressian** [comment]
@travis-tap-3d according to their documentation it should return 204 not 200. https://docs.bunny.net/reference/pullzonepublic_purgecachepostbytag . It seems their code doesn't follow their doc. Today I will change the code to consider 200 also as successful.

**Comment by ayeressian** [comment]
@travis-tap-3d fixed please use v1.1.2

**Comment by travis-tap-3d** [comment]
Using curl I was able to diagnose my issue this morning to get the proper status for purge. I was incorrectly using the zone name and not the integer “zone Id” (found in the url when viewing the zone). I also was using a token I created 
 for the zone as the access key vs my overall account API key. Perhaps the readme could be updated to be more clear or with example values?
